src/main/java/com/store/Fruit.java

```diff
@@ -0,0 +1,45 @@
+package com.store;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@Data
+@NoArgsConstructor
+public class Fruit {
+    private TypeFruit typeFruit;
+    private int shelfLife;
+    private String deliveryDate;
+    private int price;
+
+    public Fruit(TypeFruit typeFruit, int shelfLife, int price) {
```

Comment:
I`m not sure that you really understant for which purpose do you use lombok annotation

Reply:
I understant. The plugin was not installed in the settings IDEA.  I couldn't get access to getters. 
@Data - make getters/setters and

Reply:
I use my method to initialize the class field
    this.deliveryDate = setDeliveryDate();